Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_InteractableObject.cs

```diff
@@ -95,6 +106,19 @@ public enum AllowedController
         private bool previousIsGrabbable;
         private bool forcedDropped;
 
+        public bool CheckHideMode(bool default_mode, ControllerHideMode override_mode)
```

Comment:
parameter names should be pascal case not snake case.

so `defaultMode` `overrideMode` and not `default_mode` `override_mode`